route/route.go

```diff
@@ -602,6 +606,14 @@ func (r *Router) processEvent(ev *types.Event, reqID interface{}) error {
 		IsRoot:  isRootSpan(ev, r.Config),
 	}
 
+	if r.incomingOrPeer == "incoming" {
+		if span.Data["meta.signal_type"] == "log" {
+			r.Metrics.Count("bytes_received_logs", span.GetDataSize())
```

Comment:
are there any performance considerations around GetDataSize or is it returning a value that's already been calculated?

Reply:
That's a good point. I changed it to only track it when opamp is enabled like what we talked about